sections/sprint-1/experiment.tex

```diff
@@ -47,76 +47,116 @@ \subsection{Setup}
 \subsection{Analyzing the data}
 In \autoref{app:experiment} the data for the experiment can be seen.
 For each test the average grid was calculated, and the minimum and maximum values for $x$, $y$ and $z$ were found.
-Likewise, the average deviation was calculated from the actual grid to the average grid to get a better view of the deviation.
 The min and max values were noted, as they may be useful for future analysis.
 The $z$ values were very inconsistent throughout the entire test, and due to this and the z-axis being largely irrelevant to the game, we will not focus on analyzing them.
+Because it was decided to only calculate the deviance between the actual position and the measured position in the xy-plane.
```

Comment:
```suggestion
Because of this, it was decided to only calculate the deviance between the actual position and the measured position in the $xy$-plane.
```